src/openforms/forms/admin/form.py

```diff
@@ -39,6 +39,57 @@ class FormStepInline(OrderedTabularInline):
     extra = 1
 
 
+class FormDeletedListFilter(admin.ListFilter):
+    title = _("is deleted")
+    parameter_name = "deleted"
+
+    def __init__(self, request, params, model, model_admin):
+        super().__init__(request, params, model, model_admin)
+
+        self.request = request
```

Comment:
didn't notice that in the submissions filter, but `self.request` does not seem to be used anywhere - can this be removed?